.github/workflows/publish-pypi.yml

```diff
@@ -21,6 +21,14 @@ jobs:
       with:
         python-version: '3.12'
 
+    - uses: actions/setup-node@v4
```

Comment:
Yes, this is the right idea. The built static files are only needed by people who are installing the app via pypi, so this will create the files they need and we just need to make sure that they end up in the built package and will get copied by `collectstatic` by projects.